base/random.jl

```diff
@@ -1384,7 +1406,7 @@ function randexp_unlikely(rng, idx, x)
 end
 
 """
-    randn!([rng=GLOBAL_RNG], A::AbstractArray) -> A
+    randn!([rng::AbstractRNG=GLOBAL_RNG,] A::AbstractArray) -> A
```

Comment:
My 2c, I find it more readable by putting the comma outside of brackets, to separate clearly the arguments, optional or not.